modulefiles/obsproc_hera.lua

```diff
@@ -2,12 +2,15 @@ help([[
 Load environment to build obsproc on Hera
 ]])
 
-load("cmake/3.20.1")
+prepend_path("MODULEPATH", pathJoin("/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-" .. (os.getenv("spack_stack_ver") or "None"), "/envs/" .. (os.getenv("spack_env") or "None"), "-dev-rocky8/install/modulefiles/Core"))
```

Comment:
```suggestion
prepend_path("MODULEPATH", os.getenv("spack_stack_mod_path"))
```
`export spack_stack_mod_path=" .... /install/modulefiles/Core"` in `build.hera.ver` and `run.hera.ver`.  
Yes, that will be repetition, but that is already true with all the versions that are in `run.spack.ver` and `build.spack.ver`.  Ideally, we should have a single `spack.ver` and source it.  Setting a version is ok even if it is not used in `build` or `run`.  E.g. having `prod_util_ver` in build step or having `cmake_ver` in run, is harmless.
For the future.  @DavidHuber-NOAA thoughts?

Reply:
Sounds like a great idea.  Opened #2844.

Reply:
Suggested modifications made @ e097418